package.json

```diff
@@ -21,6 +21,7 @@
     "test-release": "npx semantic-release --dry-run",
     "test": "jest",
     "test-with-coverage": "jest --coverage",
+    "testWithCoverage": "jest --coverage",
```

Comment:
what's this sneaky guy for when we have `test-with-coverage`?

Reply:
I think you might be better at remembering which repo uses the dashed commands and which one the snake case ones so this way no one has to change their habits